extractorapp/templates/extractor-email-ack-template.tpl

```diff
@@ -5,4 +5,4 @@ If you have not requested an extraction, please just ignore this message.
 This e-mail is sent automatically, any answer will be ignored.
 
 ---
-Sent by geOrchestra (https://georchestra.mydomain.org/)
+Sent by geOrchestra ({publicUrl})
```

Comment:
trailing slash ?

Reply:
No trailling slash but the link is valid. We can also add a slash in template : 
```
Sent by geOrchestra ({publicUrl}/)
```